sdk-base/src/main/java/com/mapbox/maps/plugin/viewannotation/ViewAnnotationPlugin.kt

```diff
@@ -67,7 +67,7 @@ interface ViewAnnotationPlugin: MapPlugin, ContextBinder, OnCameraChangeListener
    *
    * @return [View] if view was found and NULL otherwise.
    */
-  fun getViewAnnotationByMarkerId(markerId: String): View?
+  fun getViewAnnotationByFeatureId(featureId: String): View?
```

Comment:
should we update description?
```
Find [View] by feature id if it was specified as part of [ViewAnnotationOptions.featureIdentifier]
```

same below

Reply:
fixed